addon-test-support/@ember/test-helpers/dom/-is-select-element.ts

```diff
@@ -0,0 +1,14 @@
+import { isDocument } from './-target';
+
+/**
+  @private
+  @param {Element} element the element to check
```

Comment:
```suggestion
  @param {Element|Document} element the element to check
```

Reply:
I think differing here is good, we don't _really_ want folks to pass us `Document`.